app/src/main/java/org/systers/mentorship/remote/requests/RegisterRequest.kt

```diff
@@ -16,5 +16,7 @@ data class RegisterRequest (
         val username: String,
         val email: String,
         val password: String,
-        @SerializedName("terms_and_conditions_checked") val hasAcceptedTermsAndConditions: Boolean
+        @SerializedName("terms_and_conditions_checked") val hasAcceptedTermsAndConditions: Boolean,
+		@SerializedName("need_mentoring") val needsMentoring: Boolean,
```

Comment:
Fix indentation please

Reply:
Fixed in the latest commit. Nice catch! It showed up as a well formatted string in my IDE, so it must have slipped past me. I have formatted it properly now.